### ISSUE TITLE
Unexpected behavior change from 3.6 to 4.1 for `validate()`

### ISSUE THREAD

**Issue by lundibundi** [omichelsen/compare-versions#50]
After updating to the newest version I have noticed the following:

```
// Version `compare-versions@3.6.0`:
compareVersions.validate('=1.0') // -> false

// Version `compare-versions@4.1.1`:
compareVersions.validate('=1.0') // -> true
```

is that expected?

**Comment by omichelsen** [comment]
That is not intentional. I will look into a fix, thanks for reporting.

**Comment by omichelsen** [comment]
Published v4.1.2 to fix this issue.

**Comment by lundibundi** [comment]
Great, thanks for a swift fix.

**Comment by lundibundi** [comment]
@omichelsen looks like a wrong version was published as 4.1.2.
After manually loading a tar from https://registry.npmjs.org/compare-versions/-/compare-versions-4.1.2.tgz I got 
```js
...
  compareVersions.validate = function (version) {
    return typeof version === 'string' && semver.test(version);
  };
...
```
in the package.
